### ISSUE TITLE
ConRO has been blocked

### ISSUE THREAD

**Issue by Gavardo** [Vae2009/ConRO#22]
Getting the Error of "ConRO has been blocked from an action only available to the Bizzard UI. You can disable this addon and reload the UI" I've been getting this randomly in some BG's as well as some instances such as the Brackenhide Hollow dungeon.
![Screenshot_232](https://github.com/Vae2009/ConRO/assets/10770559/107e71f3-7dcb-4a94-b733-0f7b28e2bfe0)

**Comment by Xiebinrui** [comment]
I get this as well. Frustrating. Happens pretty much any instance you zone into (or *back* into). If you're world questing and zone into a dungeon/raid, it happens. If I "fix" it in the dungeon/raid and then go back out to the world, it can happen then, too. But I mostly feel it when stepping into a dungeon. The first pull and I'm having to manually click my action buttons w/ my mouse and telling people, "One sec, let me fix this addon again..."

Basically, if you choose Disable and then manually reenable it and reload the UI, it works for that instance. Then when you go outside of that instance, or into another instance, it pops back up again and you have to repeat the process.

I love the addon, though.

Thanks!

**Comment by Vae2009** [comment]
This is an issue that is typically blamed on ConRO as I include Ace3 libraries within my addon. So when another addon that is the issue uses Ace3 and then errors ConRO will take the blame or another addon that has updated libraries.

I have a list of addons from another user I am testing today. If you would Screenshot your addon list. I may shed a little light on this issue.

**Comment by Vae2009** [comment]
I am not finding a conflict on my system so it maybe a result of old settings. I cannot say for sure. Best I can offer is you test it out with going to a dungeon causing the issue and load only ConRO then add other addons until the issue returns.

**Comment by Xiebinrui** [comment]
Oh, OK, thank you. That makes sense. I’ll try that out and see if it’s
another addon causing conflicts.

Not in front of my computer at the moment, but I know I’m using Scrap, DBM,
Details and GTFO. There may be one or two others.

Thank you!

**Comment by Xiebinrui** [comment]
Just thought I'd post a follow-up. I think I may have isolated the conflicting addon (for me) as either Plater or Threat Plates (possibly both; I think I've noticed the error popping up with either of those running [I don't run both simultaneously, since they both have similar functionality concerning replacing the default nameplates]).

**Comment by Xiebinrui** [comment]
![Screenshot 2024-01-21 184732](https://github.com/Vae2009/ConRO/assets/156253748/c6c084eb-7412-488b-8c67-6d2e032894be)
![Screenshot 2024-01-21 184749](https://github.com/Vae2009/ConRO/assets/156253748/8747d0a7-096a-4f6a-af9c-e0d1c27d122f)

Just sharing screen shots of my addons, per request.

**Comment by Xiebinrui** [comment]
Update to say that I've had it come up again, even with Plater and Threat Plates disabled, so it's something else. DBM, perhaps? Not positive. I've recently discovered that I don't need to disable the addon and then re-enable it; all I actually have to do is just do a /console reloadui and that seems to do it. I've just been getting in the habit of doing that each time I enter a dungeon.

It's been happening during questing, though, too, after campaign quests w/ cutscenes. It reverts to popping up the error once the game reloads following the cutscene.

DBM, Details and GTFO are the only other things I'm running regularly, per my screen shots above.
